app/services/notification_service.rb

```diff
@@ -0,0 +1,13 @@
+class NotificationService
+  def self.notify(push_token, message, data)
+    params = {
+      app_id: ENV['PUSH_APP_ID'],
```

Comment:
You should move this kind of service configuration to the class initializer

```
def initializer(push_token, message, data)
    OneSignal::OneSignal.api_key = ENV['PUSH_APP_ID']
    @push_token = push_token
    @message = message #since this message is always the same I think you can remove it from params, it's up to you
    @data = data
end
```